apps/etterlatte-behandling/src/main/kotlin/behandling/foerstegangsbehandling/FoerstegangsbehandlingAggregat.kt

```diff
@@ -98,7 +99,8 @@ class FoerstegangsbehandlingAggregat(
             kommentar = kommentar,
             begrunnelse = begrunnelse,
             lagretBehandling = lagretBehandling,
-            hendelser = hendelser
+            hendelser = hendelser,
+            behandlingDao
```

Comment:
hadde likt behandlingDao = behandlingDao for å være konsekvent

Reply:
haha, helt enig.